sp800-63-3/sec1_2_introduction.md

```diff
@@ -61,46 +61,32 @@ The following table states which sections of this document are normative and whi
 
 Not all digital services require authentication or identity proofing; however, this guidance applies to all such transactions for which digital identity or authentication are required, regardless of the constituency (e.g. citizens, business partners, government entities).
 
-Transactions not covered by this guidance include those that are associated with national security systems as defined in 44 U.S.C. § 3542(b)(2). Private-sector organizations and state, local, and tribal governments whose digital processes require varying levels of assurance may consider the use of these standards where appropriate.
+Transactions not covered by this guidance include those associated with national security systems as defined in 44 U.S.C. § 3542(b)(2). Private sector organizations and state, local, and tribal governments whose digital processes require varying levels of assurance may consider the use of these standards where appropriate.
```

Comment:
No need for hyphen as "private" is an adjective, not a noun. This phrase isn't conventionally hyphenated.